pyproject.toml

```
@@ -62,3 +65,53 @@ version = { attr = "integreat_compass.__version__" }
 
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
+
+[tool.pylint.main]
+jobs = 0
+load-plugins = [
+    "pylint_django",
+    "pylint.extensions.code_style",
+    "pylint.extensions.comparetozero",
+    "pylint.extensions.comparison_placement",
+    "pylint.extensions.consider_ternary_expression",
+    "pylint.extensions.docparams",
+    "pylint.extensions.emptystring",
+    "pylint.extensions.for_any_all",
+]
+enable = [
+    "consider-using-augmented-assign",
+]
+disable = [
+    "consider-using-namedtuple-or-dataclass",
+    "consider-using-tuple",
+]
+ignore-paths = [
+  ".venv",
+  "node_modules",
+  "build",
+]
+extension-pkg-whitelist = ["lxml"]
+django-settings-module="integreat_cms.core.settings"
+
+[tool.pylint.design]
+max-args = 7
+max-attributes = 7
+max-branches = 12
+max-locals = 15
+max-parents = 7
+
+[tool.pylint.messages_control]
+disable = [
+    "invalid-name",
+    "line-too-long",
+    "too-many-lines",
+    "missing-module-docstring",
+    "no-member",
+    "unsupported-binary-operation",
+]
+enable = [
+    "useless-suppression"
+]
+
+[tool.pylint.reports]
+output-format = "colorized"
```

```suggestion
output-format = "colorized"

```
I think new lines at the end of files is always a good practice